label_studio/ml_model_providers/models.py

```diff
@@ -6,19 +6,27 @@
 from django.utils.translation import gettext_lazy as _
 
 
-class ModelProviderConnection(models.Model):
-    class ModelProviders(models.TextChoices):
-        OPENAI = 'OpenAI', _('OpenAI')
+class ModelProviders(models.TextChoices):
+    OPENAI = 'OpenAI', _('OpenAI')
+    AZURE = 'Azure', _('Azure')
```

Comment:
can we name this AZURE_OPENAI? There are other offerings from Azure, and it'd be nice not to have to go back and change this in various places if/when we add support for other azure services

Reply:
Yep makes sense